geometry_msgs/Point position_max_limits
geometry_msgs/Point position_min_limits
float64 orientation_limit_angle
